### PR TITLE
Feat/initial version

### PR DESCRIPTION
**Pull request by vivekatsuperset**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up JDK 1.8
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
```
Comment (sethyates) [inline .github/workflows/ci.yml]: Why are we using an obsolete version of Java?

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+      - name: Replace Macros
+        run: |
+          grep -rl "__PUBLISHER__" settings.xml | xargs sed -i s@"__PUBLISHER__"@"$PUBLISHER"@g
+          grep -rl "__TOKEN__" settings.xml | xargs sed -i s@"__TOKEN__"@"$TOKEN"@g
+        env:
+          PUBLISHER: ${{ github.actor }}
+          TOKEN: ${{ github.token }}
+
+      - name: Compile with Maven
```
Comment (sethyates) [inline .github/workflows/ci.yml]: We'll need to integrate this into winch to provide version bumping etc

File: .github/workflows/ci.yml (hunk 1 above)
Comment (vivekatsuperset) [reply .github/workflows/ci.yml]: If it was just for us, I would've used 11 but I worry about companies (especially big ones) who may use this and they might still be on JDK1.8.  I can change it easily though if you feel strongly about it.

File: .github/workflows/ci.yml (hunk 2 above)
Comment (vivekatsuperset) [reply .github/workflows/ci.yml]: Yes, that is next on the list to do

File: .github/workflows/ci.yml (hunk 1 above)
Comment (sethyates) [reply .github/workflows/ci.yml]: Is it possible to publish multiple versions?

Comment (sethyates) [reply .github/workflows/ci.yml]: I know for example with Scala, you can publish one for say 2.11 and one for 2.12 of Scala.

File: pom.xml [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+      <id>central</id>
+      <url>https://repo1.maven.org/maven2</url>
+      <releases><enabled>true</enabled></releases>
+      <snapshots><enabled>true</enabled></snapshots>
+    </repository>
+    <repository>
+      <id>github-moonraker</id>
+      <name>Moonraker Maven Repo</name>
+      <url>https://maven.pkg.github.com/deklareddotcom/moonraker</url>
+    </repository>
```
Comment (sethyates) [inline pom.xml]: Also, it would be great if we didn't use habu repos.

File: .github/workflows/ci.yml (hunk 2 above)
Comment (sethyates) [reply .github/workflows/ci.yml]: Created a PR with that and also updated winch with your commands.

**Pull request merged**